### DISCUSSION TITLE
import rss filters

### DISCUSSION THREAD

**Discussion by Balykoo** [Novik/ruTorrent#2270] [Q&A] [answered]
Hello,

Good job, I really like your web UI !

I have a little question, How can I import rss filters from v3.8 to v3.10 ?

I have set up an http auth with user/password.

Thanks for your work !

**Comment by stickz** [comment, marked as answer]
You can copy the `urlrewriterules.dat` file and the `rss` folder in your user's profile path. Mine is located here:
`/var/www/rutorrent/share/users/stickz/settings`
